README.md

```diff
@@ -178,7 +178,7 @@ Here is a non exhaustive list of open-source projects using faster-whisper. Feel
 
 * [whisper-ctranslate2](https://github.com/Softcatala/whisper-ctranslate2) is a command line client based on faster-whisper and compatible with the original client from openai/whisper.
 * [whisper-diarize](https://github.com/MahmoudAshraf97/whisper-diarization) is a speaker diarization tool that is based on faster-whisper and NVIDIA NeMo.
-* [whisper-standalone-win](https://github.com/Purfview/whisper-standalone-win) contains the portable ready to run binaries of faster-whisper for Windows.
+* [whisper-standalone-win](https://github.com/Purfview/whisper-standalone-win) Standalone CLI executables of faster-whisper for Windows, Linux & Mac OS X. 
```

Comment:
The correct name is macOS, Mac OS X is the old name. Ref: https://en.wikipedia.org/wiki/MacOS.

Reply:
Done. Thx.